service/grails-app/views/license/_license.gson

```diff
@@ -24,7 +24,5 @@ if ( controllerName == 'license' && actionName.toLowerCase() == 'show' ) {
 
 // The majority of this template is inherited from the parent template. Just add the extensions.
 json g.render(license, [includes: ['type', 'orgs', 'amendments'], expand: ['type', 'orgs', 'amendments'], excludes: ['alternateNames']]) {
```

Comment:
I think it should be specific includes on the licenseCore template i.e. includes: [''] and on this template we should only use excludes and exclude everything that is included by the licenseCore template. That means the least amount of changes are necessary when we change the model going forward.

Reply:
I made specific includes in licenseCore template but got problems with exclude stuff from licenseCore in the license template, errors like 
`grails.plugin.json.builder.DefaultJsonGenerator.writeIterator(DefaultJsonGenerator.java:406`
Also it seems more logic to me to deal only in the licenseCore template with fields we add or remove from LicenseCore and in the license template with fields we handle in License. 
So I decided to also use specific includes in the license template, and that seems to work well. And it's easy to remember: If you change sth in LicenseCore, look at the related template (and only to that) and if you change sth in License adjust (only) the license template accordingly.
Does this sound reasonable to you?

Reply:
> Also it seems more logic to me to deal only in the licenseCore template with fields we add or remove from LicenseCore and in the license template with fields we handle in License.

I agree totally. Which is why I suggested to ignore the ones in the core in the main license template. That way all that's left is the extensions and they would get rendered automatically without needing any changes. TBH, I'm not overly bothered and if this works then I'm happy with that.